features/gezag-minderjarige/README.md

```diff
@@ -0,0 +1,55 @@
+- De vrouw uit wie het kind geboren is;
+- De vrouw die het kind heeft geadopteerd; 
+- De vrouw die met de moeder is getrouwd of een geregistreerd partnerschap heeft als het kind tijdens het huwelijk of geregistreerd partnerschap geboren wordt. Om zwanger te worden is gebruik gemaakt van een onbekende zaaddonor, zoals beschreven in de Wet donorgegevens kunstmatige bevruchting; 
+- De vrouw die het kind heeft erkend;
+- De vrouw van wie het ouderschap gerechtelijk is vastgesteld.
+
+De vader van een kind is volgens de wet:
+- De man die met de moeder is gehuwd of een geregistreerd partnerschap heeft, als het kind tijdens het huwelijk of geregistreerd partnerschap geboren wordt;
+- De man die het kind heeft erkend; 
+- De man die het kind heeft geadopteerd;
+- De man van wie het ouderschap gerechtelijk is vastgesteld. 
+
+Met niet-ouder bedoelen we:
+- de partner van één van de ouders
+- een voogd.
+
+**Wat is voogdij?**
+Voogdij is het gezag van niet-ouders. Iedereen die 18 jaar of ouder is, niet onder curatele staat en geen geestelijke stoornis heeft, kan voogd worden. Ook Bureau Jeugdzorg kan als voogd worden benoemd.
+
+Wanneer krijgt een kind een voogd?
+- wanneer de ouders zijn overleden;
+- wanneer de ouders geen gezag (meer) mogen hebben.
+
+Een kind kan automatisch of via de rechter een voogd krijgen:
+- Een gezaghouder met gezamenlijk gezag wordt automatisch voogd als de ouder overlijdt of onder curatele staat. Zie bevoegdheid-tot-gezag-feature.
+- Vaak benoemt de rechter een voogd in een gerechtelijke uitspraak. Ook ouders kunnen in hun testament of door een aantekening in het gezagsregister één of twee personen als voogd aanwijzen. De voogdij begint pas als de ouders overleden zijn, het kind nog geen 18 jaar is en de voogd bij de griffie van de rechtbank verklaard heeft dat de voogdij wordt aanvaard. Als de gevraagde persoon toch geen voogd wil zijn, bepaalt de rechter wie de voogd wordt. Zie gerechtelijke-uitspraak-feature.
+
+**Leeswijzer features**
+
+Gezag wordt bepaald door afleidingsregels (rules) in onderstaande volgorde toe te passen:
+1. Rules waarmee wordt vastgesteld dat er geen gezag is (zie de geen-gezag.feature)
+2. Rules waarmee wordt vastgesteld dat het gezag niet te bepalen is (zie de gezag-niet-te-bepalen.feature)
+3. Rules waarmee wordt vastgesteld of er een geldige gerechtelijke uitspraak is over gezag, en of deze wordt herzien door adoptie, een reparatiehuwelijk of ontkenning (zie de gerechtelijke-uitspraak.feature) 
+4. Rules waarmee het gezag wordt bepaald van een minderjarige:
+    - met twee ouders met een relatie met elkaar (zie de twee-ouders-met-relatie.feature)
+    - met twee ouders zonder een relatie met elkaar (zie de twee-ouders-geen-relatie.feature)
+    - met één ouder (zie de één-ouder.feature)
+    - zonder ouders (zie de geen-ouder.feature)
+5. Rules waarmee het gezag wordt bepaald wanneer gezaghouder(s) niet bevoegd of overleden zijn (zie de bevoegdheid-tot-gezag.feature).
```

Comment:
Voorzet voor de graph.

```mermaid
graph TD
A[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/gezag-niet-te-bepalen.feature'>1. geen gezag</a>]
A --> B[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/gezag-niet-te-bepalen.feature'>2. gezag niet te bepalen</a>]
B --> C[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/gerechtelijke-uitspraak.feature'>3. gerechtelijke uitspraak</a>]
C --> D[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/twee-ouders-met-relatie.feature'>4.a minderjarige met twee ouders met een relatie met elkaar</a>]
C --> E[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/twee-ouders-geen-relatie.feature'>4.b minderjarige met twee ouders zonder een relatie met elkaar</a>]
C --> F[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/een-ouder.feature'>4.c minderjarige met één ouder</a>]
C --> G[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/geen-ouder.feature'>4.d minderjarige zonder ouders</a>]
D --> H[<a href='https://github.com/BRP-API/brp-api-gezag/blob/main/features/specs/gezag-minderjarige/bevoegdheid-tot-gezag.feature'>5. ouder of ouders niet bevoegd of overleden</a>]
E --> H
F --> H
G --> H
```

Aantal limitaties:
- links zijn absoluut en werken dus alleen in main op GitHub
- haakjes kunnen niet worden gebruikt, bijvoorbeeld in "ouder(s)".

Te bespreken of het netto wenselijk is.